Story-Chief/frontend-test-zenowatteeuw
# Feature/search field

## Description

Added a search field to the calendar. Users are now able to filter events based on their title and description.

## Closes issue(s)



## Testing Instructions

1. Open the calendar
2. Type a term in the search field.
3. Note that only events that have that term in their title or description are now visible.

## Screenshots
![afbeelding](https://user-images.githubusercontent.com/24605774/215824490-78a4c1c5-c969-4bc0-801e-f42cccdb2b5a.png)
